### PR TITLE
add ability to edit election directly

### PR DESCRIPTION
**Pull request by benadida**

### CODE REVIEW

File: src/screens/BuildElectionScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  return (
+    <Screen>
+      <Main>
+        <MainChild maxWidth={false}>
+          <form>
+            <textarea
+              value={JSON.stringify(election, undefined, 4)}
+              cols={80}
+              rows={50}
+              onChange={handleChange}
```
Comment (beausmith) [inline src/screens/BuildElectionScreen.tsx:39]: Do you want to actually save over the existing election config as the user makes edits? Might make more sense to use state to maintain the value of the edited version until the user wishes to "save" over the existing election config. On save would be where we'd do more advanced error checking (beyond JSON.parse), or provide a diff for the user to compare, etc. before actually updating the election config.

File: src/screens/DashboardScreen.tsx [hunk 2]
```diff
@@ -112,6 +116,10 @@ const DashboardScreen = ({
         <MainChild maxWidth={false}>
           <Prose maxWidth={false}>
             <h1>Pre-Election Actions</h1>
+            <h2>Define Election</h2>
```
Comment (beausmith) [inline src/screens/DashboardScreen.tsx:119]: "Define", "Build", "Create", "Edit", "Configure", "Modify"… we should be consistent. 

I suggest "Edit" if a value exists, "Create" if no value exists… though I don't think you can get to this screen without a value, so "Edit" seems most obvious to me.

File: src/screens/BuildElectionScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            />
+            <br />
+            <Button onClick={done}>Save</Button>
+          </form>
+        </MainChild>
+      </Main>
+      <MainNav title="Build Election">
+        <Button small onClick={done}>
+          Dashboard
+        </Button>
```
[the remark this answers is not in the document]
Comment (benadida) [reply src/screens/BuildElectionScreen.tsx:49]: this one is for consistency of layout with other screens.

File: src/screens/BuildElectionScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  const handleChange = (evt: React.ChangeEvent<HTMLTextAreaElement>) => {
+    try {
+      const newElection = JSON.parse(evt.target.value)
+      if (newElection && setElection) {
+        setElection(newElection)
+      }
+    } catch {
+      // es-lint-disable-next-line no-empty
+    }
+  }
```
[the remark this answers is not in the document]
Comment (benadida) [reply src/screens/BuildElectionScreen.tsx:28]: I should probably split state-saving from parsing.

**Pull request merged**